packages/components/molecules/f-skeleton-loader/src/components/skeletons/Rating.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+    <div
+        :class="$style['c-rating']">
+        ★★★★★★
+    </div>
+</template>
+
+<script>
+export default {};
+</script>
+
+<style lang="scss" module>
+
+.c-rating {
+     color: $skeleton-loader-bone-background-color;
+     font-size: spacing(x2);
```

Comment:
```suggestion
     @include font-size(body-l, false);
```

We usually use [font-size mixin](https://github.com/justeat/f-utils/blob/10f1daa6b77bd97a1b6083c2cfd2600f5ee2febc/src/scss/mixins/_type.scss#L20) for font-sizes but as this is for size of the stars I am not sure if it make sense to change